test/parallel/test-fs-watchfile.js

```diff
@@ -19,8 +19,74 @@ assert.throws(function() {
   fs.watchFile(new Object(), function() {});
 }, /Path must be a string/);
 
-// Test ENOENT. Should fire once.
-const enoentFile = path.join(fixtures, 'empty', 'non-existent-file');
+const enoentFile = path.join(fixtures, 'non-existent-file');
+
+function customCloneObject(obj) {
+  const res = Object.create(null);
+  Object.keys(obj).forEach(function(key) {
+    res[key] = obj[key];
+  });
+  return res;
+}
+
+function areEmptyAndEqual(curr, prev) {
+  // create copies of arguments without the prototype information, because
+  // assert.deepStrictEqual compares the prototypes as well.
```

Comment:
@bnoordhuis Actually we are comparing `expected` against `curr` and `prev`. `expected` is just an object literal. So, the prototypes will be different.